rosbag2/src/rosbag2/writer.cpp

```diff
@@ -93,4 +97,12 @@ void Writer::write(std::shared_ptr<SerializedBagMessage> message)
   storage_->write(converter_ ? converter_->convert(message) : message);
 }
 
+bool Writer::should_split_bagfile() const
```

Comment:
when will this function be called?

Reply:
See the WIP PR https://github.com/ros2/rosbag2/pull/158 - this method will be called in a later commit after each write to determine if the current DB should be closed and a new one open.